examples/Makefile

```diff
@@ -0,0 +1,91 @@
+# ----------------------------------------------------------- -*- mode: sh -*-
+# List of targets to build
+# ----------------------------------------------------------------------------
+
+TARGETS += example
+
+# ----------------------------------------------------------------------------
+# Top level targets
+# ----------------------------------------------------------------------------
+
+.PHONY: build install clean distclean mostlyclean
+
+build:: $(TARGETS)
+
+install:: build
+
+clean:: mostlyclean
```

Comment:
That's how I learned about the existence of double-colon rules))  Although I couldn't understand what they are for in this particular case - I guess it's just a boilerplate, as well as many other unnecessary things in this makefile

Reply:
YMMV but for me it is: if something can be a double colon rule, it probably should be. Gives visual hint and avoids diff noise when/if it is actually used as such. But yes. It is autogenerated makefile that avoids all my personal pet peeves - and probably not appropriate for this particular case -> sheared all fluff.